core/src/main/scala/scalacss/Renderer.scala

```diff
@@ -121,6 +126,15 @@ object StringRenderer {
       mqEnd    = _         => sb append '}',
       kfsEnd   = _         => sb append '}',
       kfEnd    = _         => sb append '}',
+      ff       = fontface  => {
+        sb append s"""@font-face {font-family:"${fontface.fontFamily}";src:"""
+        for ((s, i) <- fontface.src.zipWithIndex) {
+          if (i == fontface.src.size-1) sb append s
+          else sb append s"""$s,"""
+        }
+
+        sb append s"""font-stretch:${fontface.fontStretch};font-style:${fontface.fontStyle};font-weight:${fontface.fontWeight};}unicode-range:${fontface.unicodeRange};} """
```

Comment:
Here there is a lot of duplication with the `ff = fontface => {` below.
I think it would be much better for the engine to do as much work as possible and then these interfaces be as dumb as possible (which also ensures that certain types of errors don't happen in anyone's plugins if they write one). For example, there should be a function to render each key-value pair like `("font-stretch", "expanded")` like how the CSS works.